examples/systemd/kanidmd.service

```diff
@@ -0,0 +1,20 @@
+# You should not need to edit this file. Instead, use a drop-in file by running:
```

Comment:
I think there are a lot of hardenings missing from https://github.com/kanidm/kanidm/blob/master/platform/opensuse/kanidmd.service

Reply:
The challenge was that a lot of them weren't available on Ubuntu/Debian because their systemd versions weren't as bleeding-edge as SUSE seem to be.

Reply:
Ahhhhhh! All good then :)